scripts/init.sh

```diff
@@ -25,13 +26,16 @@ rm -rf ../.clasp.json_prod
 rm -rf ../.clasp.json_test
 cd ../dist
 
-echo "Create project $name"
+echo "Creating $name"
```

Comment:
The echo above was using "Preparing" so though of keeping the verbiage consistent here :)